Sources/View+Rect.swift

```diff
@@ -57,7 +57,7 @@ extension UIView {
 import AppKit
 
 extension NSView {
-    func getRect(keepTransform: Bool) -> CGRect {
+    public func getRect(keepTransform: Bool) -> CGRect {
```

Comment:
Same here, I would move immediately `getRect` and `setRect` in NSView+PinLayout.swift

Reply:
Done